services/venice-controller/src/main/java/com/linkedin/venice/controller/DeferredVersionSwapService.java

```diff
@@ -64,6 +65,8 @@ public class DeferredVersionSwapService extends AbstractVeniceService {
   private Set<String> stalledVersionSwapSet = new HashSet<>();
   private Map<String, Integer> failedRollforwardRetryCountMap = new HashMap<>();
   private static final int MAX_ROLL_FORWARD_RETRY_LIMIT = 5;
+  private static final Set<VersionStatus> versionSwapCompletionStatuses = new HashSet<>();
```

Comment:
assign the values during declaration. you can use
`statuses = Utils.setOf(status1, status2...`